developmentseed/bioacoustics-frontend
# Setup NextJS infrastructure

Implements #2

- Set up front-end, bootstrapped from NextJS
- Configure ESlint rules

**How to test**

- check-out and install locally
- run `yarn dev`
- open the site at http://localhost:3000, where you should see the default NextJS page
